### ISSUE TITLE
Difficulties with chord transitions matrix

### ISSUE THREAD

**Issue by xhevahir** [flothesof/SongCreator#2]
From the screenshots in the readme it looks like SongCreator treats some chords as repeating--which makes sense, except when I asked the creator of Songsmith why his program does not allow for these "self-transitions," he answered in [this thread](https://social.microsoft.com/Forums/en-US/ae56773a-7bfd-4a2d-a32f-b7351a5780f0/bias-in-the-chord-transitions-model?forum=songsmith) . Maybe they should be ruled out?

**Comment by flothesof** [comment]
Good point! You are right, they should be ruled out as the thread says.
